lib/workers/repository/process/lookup/index.ts

```diff
@@ -266,8 +266,10 @@ export async function lookupUpdates(
       }
 
       res.currentVersion = currentVersion!;
-      const currentVersionTimestamp = allVersions.find((v) =>
-        versioning.equals(v.version, currentVersion),
+      const currentVersionTimestamp = allVersions.find(
+        (v) =>
+          versioning.isValid(v.version) &&
```

Comment:
Do we know why `v.version` might not be a string?

Reply:
No, but I'd be interested if you can help me get your reproduction working

Reply:
can we log a debug message when version isn't valid here?